### ISSUE TITLE
Found a bug in your code!

### ISSUE THREAD

**Issue by Kiminaze** [IndianaBonesUrMom/fivem-seatbelt#4]
For everyone having these weird "it only occurs sometimes and usually only when hitting other entities"

Line 38 looks like this: "and speedBuffer[1] > Cfg.MinSpeed "

Change it to this: "and speedBuffer[2] > Cfg.MinSpeed "

Now it works 99% of the time. (probably just some 1% GTA side effects ;)  )

Thanks for the script though! Will put it to great use!

**Comment by OhTanoshi** [comment]
This dosent fix only hitting entity's. tried both ways and the way you shows defiantly makes it more frequent and stable. but dosent fix the issue

**Comment by Kiminaze** [comment]
I suggest looking into the code a little more.

We actually fixed a lot of issues with this script and as of now it is working 100% as intended on our server.
I am kinda new to github and need to look into a lot of things, but here is what I got:

[client.zip](https://github.com/IndianaBonesUrMom/fivem-seatbelt/files/2264869/client.zip)

You need to change things like the notifications as we use a custom framework but apart from that it should be working flawlessly.

**Comment by OhTanoshi** [comment]
Yeah i noticed right off. but you also have `Citizen.CreateThread(function()
    while p01 == nil do
        TriggerEvent('ProjektZero1:getAllFunctionality', function(obj) p01 = obj end)
        Citizen.Wait(0)
    end
end)` Which idk what that does if its part of your seatbelt system or if its to do with the notifications. i havent tested it to see if it will work but yeah

**Comment by OhTanoshi** [comment]
Never mind that thats for notifications i see.

**Comment by Kiminaze** [comment]
Yea it is... our framework is in the early stages, doesn't have a proper name yet, but we try to implement in a lot of scripts to find all its kinks ...

And, does it work as expected for you? :D

**Comment by OhTanoshi** [comment]
A few modifications and making a config files and it works perfect. gonna end up working on some crash effects and such to it also now that seat belts actually have a use lol
